streamer/Cargo.toml

```diff
@@ -49,6 +49,7 @@ solana-packet = { workspace = true }
 solana-perf = { workspace = true }
 solana-pubkey = { workspace = true }
 solana-quic-definitions = { workspace = true }
+solana-rayon-threadlimit = { workspace = true }
```

Comment:
I do not think that it is intuitive that a default tokio runtime worker count is attached to a "SOLANA_RAYON_THREADS" environment variable. I'd set the default at either ` num_cpus::get()` or even just hardcode it to 8 as per Alessandro's suggestion.

Reply:
Agreed, let us define the good default worker thread count as we discussed: 8 for all three cases.

Reply:
Fair, the `solana-rayon-threadlimit` crate is a dirty hack that was put in place to mitigate "too-many-thread" issue before systematically reducing all pools.

I'll switch this to `num_cpus::get()` for the intermediate, and then we'll quickly remove that dep in a follow on PR when we replace that with whatever default is good (8 is the number we heard)